private/bufpkg/bufpolicy/digest.go

```diff
@@ -221,7 +215,7 @@ func (*digest) isDigest() {}
 
 // getO1Digest returns the O1 digest for the given PolicyConfig.
 func getO1Digest(policyConfig PolicyConfig) (Digest, error) {
```

Comment:
Pretty sure we're going to be needing this one too? it's easy enough to do it ourselves with the `bufcas` pkg too. I guess it's just a matter of making sure use use the same `DigestType` everywhere.